.circleci/src/jobs/bump_version.yml

```diff
@@ -12,15 +12,19 @@ steps:
   - restore_yarn_tools_cache
```

Comment:
Perhaps it make sense to adjust this accordingly to integreat (change enum `delivery` to boolean `prepare_delivery` param

Reply:
As this wasn't changed in integreat, when I was starting to work on this ticket, I would say out of scope, but if you think this change is important, feel free to open a ticket.